.github/CODEOWNERS

```diff
@@ -131,12 +131,12 @@
 /LICENSE @seanlip
 /NOTICE @seanlip
 /core/templates/pages/about-page/ @DubeySandeep
-/core/templates/pages/about-page/about-page.constants.ts @seanlip @BenHenning
+/core/templates/pages/about-page/about-page.constants.ts @seanlip
 /core/templates/pages/license-page/ @vojtechjelinek
-/core/templates/pages/license-page/license-page.component.html @seanlip @BenHenning
+/core/templates/pages/license-page/license-page.component.html @seanlip
 /core/templates/pages/privacy-page/ @DubeySandeep
-/core/templates/pages/privacy-page/privacy-page.component.html @seanlip @BenHenning
-/core/templates/pages/privacy-page/privacy-page.component.ts @seanlip @BenHenning
+/core/templates/pages/privacy-page/privacy-page.component.html @seanlip
```

Comment:
Wait, let's keep @BenHenning on this line please (I only excluded mainpage but might have been looking at an old version of codeowners). Thanks!

Reply:
Done.